sys/net/gnrc/network_layer/ipv6/ext/gnrc_ipv6_ext.c

```diff
@@ -85,53 +211,25 @@ static int _handle_rh(gnrc_pktsnip_t *current, gnrc_pktsnip_t *pkt)
 #endif  /* MODULE_GNRC_IPV6_EXT_RH */
 
 /**
- * @brief marks IPv6 extension header if needed.
- *   updates pkt and returns next header.
- * @param[in] current  The current header
- * @param[in,out] pkt  The whole packet
- * @return The next header
- * @return NULL on error
+ * @brief       Marks IPv6 extension header according to the given length
```

Comment:
length given by what? pkt.data.len?

Reply:
I will fix in the corresponding PR (have to look and see which one that is myself ;-)).

Reply:
See https://github.com/RIOT-OS/RIOT/pull/10234/commits/92e8b9f39330b5c71ea70e6bc18f025f9b72a48e (and https://github.com/RIOT-OS/RIOT/pull/10234#issuecomment-441191236).